### PR TITLE
Got github actions working

### LINKED ISSUE #4: Set up CI / unit testing
Make all commits automatically go through (github actions?) tests.

### PR DESCRIPTION
**Pull request by msbarry**
Fixes #4 by following instructions no the "gl" package to get it working on headless linux. I made progress on the windows build, but couldn't quite get it happy invoking bash script files from npm run scripts.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# see: https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings#per-repository-settings
+
+# Set the default behavior to use LF line endings
+* text eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.mvt binary
+*.pbf binary
```
Comment (msbarry) [inline .gitattributes:9]: I think we need at least .js, .json, and .html files to be LF line endings for tests and eslint to be happy - not sure if we should default everything to LF like this? Or selectively opt-in those file types to make the build pass.

Here are the file type counts in the repo - only png, mvt, pbf stood out as binary to me...
```
$ git ls-files | sed 's/^.*\././' | sort | uniq -c | sort -n
      1 .dockerfile
      1 .dockerignore
      1 .editorconfig
      1 .flow
      1 .gitattributes
      1 .gitignore
      1 .lock
      1 .mapbox-streets-v7
      1 .npmignore
      1 .satellite
      1 .sh
      1 .stylelintrc
      1 .topissuesrc
      1 .txt
      1 build/run-node
      1 build/run-tap
      1 src/style-spec/bin/gl-style-composite
      1 src/style-spec/bin/gl-style-format
      1 src/style-spec/bin/gl-style-migrate
      1 src/style-spec/bin/gl-style-validate
      2 .css
      2 .flowconfig
      2 .gitkeep
      2 .template
      2 .yml
      3 .ejs
      4 .tmpl
      5 .eslintrc
      9 .geojson
      9 .svg
     19 .md
     25 .mvt
     52 .glsl
     54 .html
    175 .pbf
    557 .js
   1312 .png
   1976 .json
```

Comment (nyurik) [reply .gitattributes:9]: I would seek advice of the Windows devs for this. Personally I would prefer the files to always stay "as-in-git" (I'm not a big fan of magic conversions on checkout/checkin), but I'm ok for the well known files to be autoconverted if Windows devs so desire.  Regardless, I think we can already merge this as Windows build can be done in a separate PR.

**Review by nyurik — APPROVED** [review]
awesome, thanks!  I should have read the gl lib instructions :)

**Comment by snickell** [comment]
@msbarry NICE WORK!

**Pull request merged**